setup.py

```diff
@@ -102,11 +103,13 @@ def get_version(version_file):
             # 1.11.1 fixed this issue, but to ensure expected behavior we'll pin
             # to never grab the regression version.
             "coveralls != 1.11.0",
+            "docker == 7.1.0",
```

Comment:
Why is this one `test` dependency constrained to a specific version?  If a specific version is indeed required then there should be a comment explaining why.

Reply:
Fixed in: 
- 50d2258d82d0558ff6f54d7d800009607a51548f

Reply:
Thanks.  See also [here](https://github.com/cisagov/cyhy-db/pull/5#discussion_r1797261324) in cisagov/cyhy-db#5 where I brought up the same issue.